db/schema.rb

```diff
@@ -124,6 +125,7 @@
     t.integer "previous_id"
     t.boolean "is_published", default: false
     t.string "author"
+    t.boolean "navigation", default: true
```

Comment:
This field looks like it has slipped in from another branch. This line should be removed from the schema.